### DISCUSSION TITLE
How create a new rails app with activeadmin v4

### DISCUSSION THREAD

**Discussion by inyerade** [activeadmin/activeadmin#8361] [Help]
Hi, I need some help to create a new app with activeadmin version 4. I do not know what step I miss.

# INSTALLATION

## 1. create ralis app

```
    rails _7.1.3.2_ new aa_tailwind_rails --database=sqlite3 --css=tailwind --js=importmap
```

## 2. Add AA and devise in Gemfile

```
    ...
    gem "activeadmin", "4.0.0.beta7"
    gem "devise"
   ...
```

## 3. run command to install

```
    bundle install
    rails g active_admin:install User
    rails db:migrate
    rails db:seed
```
   
## 4. Create page outside activeadmin (/admin) to check tailwind works

In `routes.rb` add  `get "dashboard", to: "application#dashboard"`

create file in `app/views/shared/dashboard.html.erb` with:

```
<h1 class="text-3xl font-bold underline">
    Hello world!
</h1>
```

Add `dhasboard` in `application_controller.rb`

```
class ApplicationController < ActionController::Base
    def dashboard
        render "shared/dashboard"
    end
end
```   
   ## 5. Run server

```
rails server
```
# RESULT

## Outside activeadmin result (OK)

![imagen](https://github.com/activeadmin/activeadmin/assets/938614/fcc1d8de-9d24-4df7-9f6a-beebdd637bda)

## Activeadmin result (NOT OK)

![imagen](https://github.com/activeadmin/activeadmin/assets/938614/46ef12b8-a0e3-4c0d-a055-4c1a439d0314)

Can anyone help?

**Comment by javierjulio** [comment]
@inyerade I believe the problem is the `--css=tailwind` flag which actually uses the tailwindcss-rails gem which we don't support currently. That may change in the future as authors of that gem are looking into engine support. In the UPGRADING guide it's [mentioned that the expectation is to use cssbundling-rails instead](https://github.com/activeadmin/activeadmin/blob/master/UPGRADING.md#from-v3-to-v4-beta). If you still prefer to [use tailwindcss-rails, there is a helpful discussion on how to do that](https://github.com/activeadmin/activeadmin/discussions/8223#discussioncomment-8274079) that you can try. Thank you.

**Comment by inyerade** [comment]
thanks @javierjulio for the clarification. I read both link and I do not understand very well the processI think I will stay to version 3 for a while.
